src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -277,8 +280,7 @@ class ChannelData {
   void UpdateStateAndPickerLocked(
```

Comment:
This function name is now ambiguous since we don't know which lock is being held. Can I suggest a comment above the name saying which lock must be held before entering the function? Or a commented ABSL_GUARDED_BY annotation (which I think we still can't use but which could at least be helpful for readability as a comment)

Reply:
I guess it was already ambiguous since there were already other gpr_mu and Mutex in the struct, so this really needs a detailed comment.

Reply:
Yeah, it's a little confusing.  In `ChannelData`, the `Locked()` suffix doesn't actually have anything to do with the mutexes; it actually indicates that a method needs to be called from within `work_serializer_`.  I've added a comment about this.

I would very much like to be able to use lock annotations in OSS, but no one's done the work to make that happen yet.  But in this particular case, they wouldn't help anyway, since lock annotations won't work for a `WorkSerializer` the way that they do for a mutex.